packages/volto-slate/src/editor/plugins/Image/render.jsx

```diff
@@ -1,22 +1,11 @@
 import React from 'react';
-import { useSelected, useFocused } from 'slate-react';
 
 export const ImageElement = (props) => {
-  const { attributes, children, element } = props;
-  const selected = useSelected();
-  const focused = useFocused();
-
-  const style = {
-    display: 'block',
-    maxWidth: '100%',
-    maxHeight: '20em',
-    boxShadow: selected && focused ? '0 0 0 2px blue' : 'none',
-  };
+  const { attributes, children } = props;
 
   return (
     <span {...attributes} style={{ display: 'inline-block' }}>
       {children}
-      <img alt="" src={element.url} style={style} />
```

Comment:
@AnshulKahar2729 We can't do this. This is not the way to fix this issue. We have to dig deeper and find why it's emitting the slate image (which we don't want) and make that it does not emmit it. By removing the markup for the slate image is not an acceptable "fix".

 Do you agree with me, right?

Reply:
Ok, I got the problem, we need to fix the function which creates extra block rather than removing it manually.